test/ventilation_test.c

```diff
@@ -17,22 +17,41 @@ void reset_to_inspiration_start();
 int get_ventilation_phase();
 int get_start_current_breath_cycle();
 
+int getCircuitPressure_validValues_returnsMeanPressure() {
+  uint32_t inspiratory[2] = {35, 53};
+  uint32_t expiratory[2] = {40, 53};
+
+  status_t status[2] = {STATUS_OK, STATUS_OK};
+
+	set_inspiratory_pressure(inspiratory, status);
+	set_expiratory_pressure(expiratory, status);
+
+	uint32_t circuit_pressure = get_circuit_pressure();
+  //The integer division of an odd number by two is the floor value
+	assertTrue(circuit_pressure == 37);
+
+  circuit_pressure = get_circuit_pressure();
+  //The integer divion of an even number by two is as expected
+	assertTrue(circuit_pressure == 53);
+
+}
+
 int startInspiration_afterOneInspiration_openInspiratoryValve() {
   //initialize the function get_respiratory_rate
-  uint32_t RR[1] = {7500000};
+  uint32_t RR[1] = {20};
   status_t status[1] = {STATUS_OK};
   set_respiratory_rate(RR, status);
 
   //initialize the function get_current_time
-  int times[1] = {RR[0] + 1};
+  int breath_cycle_duration = 60000/RR[0];
```

Comment:
Are we sure this will return an INT for all possible values of RR, and not a FLOAT? Should we cast it just to be safe?